transaction/src/model/instruction.rs

```diff
@@ -70,21 +70,36 @@ pub enum Instruction {
     },
 
     /// Creates a proof from the auth zone, by the given amount
-    CreateProofFromAuthZoneByAmount {
+    CreateProofFromAuthZoneOfAmount {
```

Comment:
Would you consider putting resource address first in these instructions for consistency with transfer etc (as per my comment in the other PR?)

Reply:
(And eg consistency with account's `create_proof_of_amount`)

Reply:
I will have no problem with this.

A much larger clean-up will be required to fix all these parameter order inconsistency.